app/src/androidTest/java/com/arygm/quickfix/MainActivityTest.kt

```diff
@@ -17,29 +14,6 @@ class MainActivityTest {
 
   @get:Rule val composeTestRule = createComposeRule()
 
-  @Test
-  fun bottomNavigation_isVisibleOnHomeScreen() {
-    composeTestRule.setContent { QuickFixTheme { QuickFixApp() } }
-
-    val testEmail = "john.doe@epfl.ch"
-    val testPassword = "verySecurePassword"
-
-    // move to log in
-    composeTestRule.onNodeWithTag("logInButton").performClick()
-    composeTestRule.mainClock.advanceTimeBy(1000L)
-    // enter email
-    composeTestRule.onNodeWithTag("inputEmail").performTextInput(testEmail)
-
-    // enter password
-    composeTestRule.onNodeWithTag("inputPassword").performTextInput(testPassword)
-
-    // click on log in
-    composeTestRule.onNodeWithTag("logInButton").performClick()
-    composeTestRule.mainClock.advanceTimeBy(500L)
-    // Verify that the bottom bar is displayed on the Home screen
-    composeTestRule.onNodeWithTag("BottomNavMenu").assertIsDisplayed()
-  }
-
```

Comment:
A new version of this test should be written at some point with a test account specifically created for that purpose